circt.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+GIT_COMMIT=093cdfe482530623fea01e1d3242af93e533ba54
+
+CIRCT_BUILD=./build-circt
+CIRCT_INSTALL=./../circt
+
+function commit()
+{
+	echo ${GIT_COMMIT}
+}
+
+while [[ "$#" -ge 1 ]] ; do
+        case "$1" in
+                --get-commit-hash)
+                        commit >&2
+                        exit 1
+                        ;;
+        esac
+done
```

Comment:
Could we get a `--help` flag similar to the configure.sh script?

Reply:
Added